cob_obstacle_distance/msg/ObstacleDistance.msg

```diff
@@ -0,0 +1,3 @@
+Header id
+float64 distance
+geometry_msgs/Point distance_vector
```

Comment:
did you investigate whether the distance provided by fcl is the same as the norm of the distance_vector?